Library/Homebrew/utils.rb

```diff
@@ -126,8 +126,8 @@ def self.install_gem_setup_path! gem, executable=gem
     ENV["PATH"] = "#{Gem.user_dir}/bin:#{ENV["PATH"]}"
 
     unless quiet_system "gem", "list", "--installed", gem
```

Comment:
Fair enough, makes sense. Interested in a more complicated but user-expectation-friendly workaround?